test/lib/usd/translators/testUsdExportSchemaApi.py

```diff
@@ -59,12 +59,29 @@ def testExportJobContextLister(self):
             'apiSchema=[testApi];chaser=[NullAPIChaser];chaserArgs=[[NullAPIChaser,life,42]];')
 
         modes = set(cmds.mayaUSDListJobContexts(im=True))
-        self.assertEqual(modes, set(["Null API", 'Bullet Physics API Support']))
+        self.assertEqual(modes, set(["Null API", 'Bullet Physics API Support', 'Scene Grinder']))
         self.assertEqual(cmds.mayaUSDListJobContexts(importAnnotation="Null API"),
                          "Imports an empty API for testing purpose")
         self.assertEqual(cmds.mayaUSDListJobContexts(importArguments="Null API"), 
             'apiSchema=[testApiIn];chaser=[NullAPIChaserIn];chaserArgs=[[NullAPIChaserIn,universe,42]];')
 
+        # The job context used for UI debugging have int startTime and float frameStride values to convert:
+        self.assertEqual(set(cmds.mayaUSDListJobContexts(ig="Scene Grinder").strip(";").split(";")), 
+            set(['importUSDZTextures=1',
+                 'preferredMaterial=standardSurface',
+                 'readAnimData=1',
+                 'startTime=12',
+                 'useCustomFrameRange=1']))
+
+        self.assertEqual(set([i.rstrip("0")
```

Comment:
Strip the trailing zeroes of the frameStride in case the precision differs per platform.